### PR TITLE
[AIRFLOW-XXX] Add Flipp to list of Airflow users

### PR DESCRIPTION
**Pull request by sethlwilson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ Currently **officially** using Airflow:
 1. [Easy Taxi](http://www.easytaxi.com/) [[@caique-lima](https://github.com/caique-lima) & [@WesleyBatista](https://github.com/WesleyBatista) & [@diraol](https://github.com/diraol)]
 1. [eRevalue](https://www.datamaran.com) [[@hamedhsn](https://github.com/hamedhsn)]
 1. [evo.company](https://evo.company/) [[@orhideous](https://github.com/orhideous)]
+1. [Flipp] (https://www.flipp.com) [@sethwilsonwishabi]
```
Comment (kaxil) [inline README.md]: It should be 
```
[Flipp](https://www.flipp.com) [[@sethwilsonwishabi](https://github.com/sethwilsonwishabi)]
```

**Review by kaxil — CHANGES_REQUESTED** [review]

**Comment by sethlwilson** [comment]
Entry format corrected

**Comment by feng-tao** [comment]
lgtm

**Comment by Fokko** [comment]
Welcome @sethwilsonwishabi

**Pull request merged**